collectors/python.d.plugin/alarms/metadata.yaml

```diff
@@ -35,19 +41,85 @@ setup:
     list: []
   configuration:
     file:
-      name: ''
+      name: python.d/alarms.conf
       description: ''
     options:
```

Comment:
Can you check that all config options are listed?

I see in the config that there are some that are not listed, we talked in slack about using 

```yaml
 options:
      description: |
        There are 2 sections:

        * Global variables
        * One or more JOBS that can define multiple different instances to monitor.

        The following options can be defined globally: priority, penalty, autodetection_retry, update_every, but can also be defined per JOB to override the global values.

        Additionally, the following table contains all the options that can be configured inside a JOB definition.

        Every configuration JOB starts with a `job_name` value which will appear in the dashboard, unless a `name` parameter is specified.
      folding:
        title: "Config options"
        enabled: true
      list:
        - name: update_every
          description: Sets the default data collection frequency.
          default_value: 5
          required: false
        - name: priority
          description: Controls the order of charts at the netdata dashboard.
          default_value: 60000
          required: false
        - name: autodetection_retry
          description: Sets the job re-check interval in seconds.
          default_value: 0
          required: false
        - name: penalty
          description: Indicates whether to apply penalty to update_every in case of failures.
          default_value: yes
          required: false
        - name: name
          description: Job name. This value will overwrite the `job_name` value. JOBS with the same name are mutually exclusive. Only one of them will be allowed running at any time. This allows autodetection to try several alternatives and pick the one that works.
          default_value: ''
          required: false
```

Because more or less it applies to most python collectors.

please check for fail2ban too (I don't recall if `example` has the same options)

Reply:
yeah was thinking silly to have the default global ones that don't really matter in that list at the top. the ones specific to the collector make most sent to be at top and then the usual global ones can go under that - would that work?

users should be seeing the most important/relevant config options first i think. 

will rejig to show what i mean

Reply:
think done here: https://github.com/netdata/netdata/pull/15510/commits/0f29133f45c0bfd5b27b2326d622868ed082f0c7

Reply:
I see, makes a bit of sense tbh, might start implementing it that way

cc to this idea @MrZammler @ilyam8 @thiagoftsm @tkatsoulas